WooCommerce/Classes/AppDelegate.swift

```diff
@@ -428,3 +440,15 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
         await ServiceLocator.pushNotesManager.handleNotificationInTheForeground(notification)
     }
 }
+
+// MARK: - Universal Links
+
+extension AppDelegate {
```

Comment:
Should this extension be private?

Reply:
Nice point! Changed in fc725e8d82689d1a62f37570239780fb8e8f2168